packages/web-components/src/components/masthead/left-nav.ts

```diff
@@ -221,7 +221,21 @@ class C4DLeftNav extends StableSelectorMixin(CDSSideNav) {
     document.addEventListener('click', this._handleClickOut.bind(this));
   }
 
-  updated(changedProperties) {
+  protected async willUpdate(changedProperties) {
+    if (changedProperties.has('expanded') && !this._importedSideNav) {
+      await Promise.all([
+        import('./left-nav-name'),
+        import('./left-nav-menu'),
+        import('./left-nav-menu-section'),
+        import('./left-nav-menu-item'),
+        import('./left-nav-menu-category-heading'),
+        import('./left-nav-overlay')
+      ]);
```

Comment:
We don't want to await each of these in series, we should see a slight perf boost by awaiting in parallel.